### PR TITLE
Nancy appweb

### PR DESCRIPTION
**Pull request by nancymic2**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

Create web-to-app and app-to-web intro doc for Chrome

- https://pr-6978-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/attribution-reporting/web-to-app-attribution/
-
-

### CODE REVIEW

File: site/en/docs/privacy-sandbox/attribution-reporting/web-to-app-attribution/index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+subhead: >
+  Learn how your Chrome web app can pass attributions to your Android app.  
+description: >
+  Learn how your Chrome web app can pass attributions to your Android app.    
+date: 2023-08-09
+authors:
+  - maudn
+---
+
+The Privacy Sandbox natively supports cross- web and app attribution, and the Attribution Reporting API allows measurement across mobile browsers and Android apps.
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/attribution-reporting/web-to-app-attribution/index.md:13]:
```suggestion
The Privacy Sandbox natively supports cross-web and app attribution, and the Attribution Reporting API allows measurement across mobile browsers and Android apps.
```

Comment (anusmitaray) [reply site/en/docs/privacy-sandbox/attribution-reporting/web-to-app-attribution/index.md:13]: Or is that not it? What is it actually trying to say? cross web-and-app?

File: site/en/docs/privacy-sandbox/attribution-reporting/web-to-app-attribution/index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## What is web-to-app attribution?
+
+If a user clicks an ad in a Chrome mobile browser, and then goes on to later make a purchase in an Android app, the Attribution Reporting API can directly attribute that conversion performed in the Android app to the ads shown in the Chrome mobile browser. That's web-to-app attribution.
+
+Similarly, if a user clicks an ad in an Android app, and then goes on to later make a purchase in a Chrome mobile browser, the Attribution Reporting API can directly attribute that conversion. That's app-to-web attribution.
+
+Note that the API records web-to-app attributions when they occur on the same device.
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/attribution-reporting/web-to-app-attribution/index.md:21]: This opens more questions for me. Does this work only if it's on the same device? What if I open an ad on an app, switch to the browser and then open the browser on a different device–with a larger screen–to make the purchase? Does that count as a conversion? How does the device switching affect the other use cases?

File: site/en/docs/privacy-sandbox/attribution-reporting/web-to-app-attribution/index.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+## How is web-to-app attribution implemented?
+
+To implement web-to-app attribution, first make sure that cross-web and app measurement is available in your web code base.
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/attribution-reporting/web-to-app-attribution/index.md:25]: Same as comment 1

File: site/en/docs/privacy-sandbox/attribution-reporting/web-to-app-attribution/index.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+To do this, when you register an event, include the `Attribution-Reporting-Eligible` header
+in your request to the reporting origin.
+
+The browser will broadcast if OS-level support is available to the reporting origin server with a dictionary structured request header.
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/attribution-reporting/web-to-app-attribution/index.md:29]:
```suggestion
The browser will broadcast if OS-level support is available to the reporting origin server with a dictionary-structured request header.
```

File: site/en/docs/privacy-sandbox/attribution-reporting/web-to-app-attribution/index.md (hunk 2 above)
Comment (anusmitaray) [reply site/en/docs/privacy-sandbox/attribution-reporting/web-to-app-attribution/index.md:21]: (Also the ideal answer is probably "the attributions are only counted when they occur on the same device", right?)

Comment (anusmitaray) [reply site/en/docs/privacy-sandbox/attribution-reporting/web-to-app-attribution/index.md:21]: https://developer.android.com/design-for-safety/privacy-sandbox/attribution#app-web-based-trigger-paths

File: site/en/docs/privacy-sandbox/attribution-reporting/web-to-app-attribution/index.md (hunk 1 above)
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/attribution-reporting/web-to-app-attribution/index.md:13]: Ya, changing :)

File: site/en/docs/privacy-sandbox/attribution-reporting/web-to-app-attribution/index.md (hunk 2 above)
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/attribution-reporting/web-to-app-attribution/index.md:21]: is this not helping? "Note that the API records web-to-app attributions when they occur on the same device." i.e. line 21?

Comment (nancymic2) [reply site/en/docs/privacy-sandbox/attribution-reporting/web-to-app-attribution/index.md:21]: Also I'm not sure now if this doc is even needed. Maybe until the move though

File: site/en/docs/privacy-sandbox/attribution-reporting/web-to-app-attribution/index.md (hunk 3 above)
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/attribution-reporting/web-to-app-attribution/index.md]: See what you think now. (updated)

**Review by anusmitaray — APPROVED** [review]
I'm happy to let it all be available on DCC for now before deciding how to organise all the ARA content as a step 2.

**Comment by nancymic2** [comment]
> I'm happy to let it all be available on DCC for now before deciding how to organise all the ARA content as a step 2.

great thanks. In that case, merging.

**Pull request merged**